src/point-set.ts

```diff
@@ -99,15 +104,15 @@ const NO_EDGE: OnEdge = {
   bottom: false
 };
 
-export function onEdge(set: PointSet, point: Point): OnEdge {
-  if (!has(set, point)) {
+export function onEdge(set: PointSet, IPoint: IPoint): OnEdge {
```

Comment:
Should not name a variable w/capital letter first, also is confusing because `IPoint` is an interface. Also creates shadowing which TS or TSLint should have been yelling about.
```suggestion
export function onEdge(set: PointSet, point: IPoint): OnEdge {
```